### ISSUE TITLE
region restore?

### ISSUE THREAD

**Issue by mrcoffee1026** [eccentricdevotion/TARDIS#132]
**Describe the feature request**
Not sure what has been causing it but I've had at least two players now where the regions around their tardis has simply vanished... so all their material remains in tact and in place but they have no permissions to do anything there as the region permitting their build or use of things just no longer exists.  Could there be a command to restore the existence of a missing region,? I've been manually putting them back, creating them along the chunk borders and keeping the naming convention but if there's any other information being stashed in the plugin, I'm not aware of it.  There's a table for chunks, but a quick look let me know this is not a comprehensive list by any means of what is actually in use per tardis so it doesn't give a lot to go on.

**Describe alternatives you've considered**
Just the approach I'm currently taking which is to recreate the worldguard region manually and assign the player in question as the member and owner of that region so they can use it again.

**Comment by eccentricdevotion** [comment]
`/tardisadmin add_regions` already exists :)

**Comment by mrcoffee1026** [comment]
ah ok i didn't know about that, and from what i can tell it looks like it restores any missing ones? so that's great.

**Comment by eccentricdevotion** [comment]
I've not used it for sometime and don't actually remember why I added it in the first place, but it basically reads the WorldGuard regions file and compares the TARDIS records to see if the appropriate region name exists in the file. If it doesn't it creates the region.

**Comment by mrcoffee1026** [comment]
ah ok.  Well when i ran it i saw it went through a whole list of user names so it likely did what i was taking a lot of effort to do manually to all the missing one... I won't really know unless someone else says they can't use anything in their tardis either though... if it was using only the worldguard region information by itself, that might not be enough since the regions were gone when I visited these areas I don't believe they would have been in the worldguard files anymore.  How they got removed in the first place though, I have no idea.  It's only me and 2 other staff that could remove the regions and none of us did, so I just don't know.  Anyway I'll get more chances to find out more about this if anyone else has tardis trouble, for now this probably solved it.  I'm hoping it checked through the database for existing TARDIS builds and compared THAT to the list of existing regions... if it was doing that, then it should have restored all of the missing ones.
